apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -742,6 +857,28 @@ static void tc_fs_vfs_seekdir(void)
 	itoa((int)offset, filename, 10);
 	TC_ASSERT_EQ("readdir", strncmp(dirent->d_name, filename, 1), 0);
 
+	/* For Negative offset in seekmountdir operations */
+	dirp = opendir(VFS_FOLDER_PATH);
```

Comment:
If dirp is not null, you should close it. please check all opendir here.

Reply:
It is been closed in below cleanup macro.

Reply:
Below is assert.
If dirp is not null, there's memory leak because opendir allocates memory